main.go

```diff
@@ -289,3 +324,65 @@ func hasLicense(b []byte) bool {
 	return bytes.Contains(bytes.ToLower(b[:n]), []byte("copyright")) ||
 		bytes.Contains(bytes.ToLower(b[:n]), []byte("mozilla public"))
 }
+
+// reLicense contains regexp to parse years in actual header
+// 2020
+// 2018-2020
+var reLicense = regexp.MustCompile(`(([\d]{4})-)?([\d]{4})`)
+
+// Year contains information about parsed header
+type Year struct {
+	creation         string
+	lastModification string
+	currentString    string
+}
+
+// parseLicenseYear returns Year representation of actual license header
+func parseLicenseYear(b []byte) (bool, *Year) {
```

Comment:
I wonder if returning bool here is needed, as you can just check if `*Year` is not nil, what do you think?
```suggestion
func parseLicenseYear(b []byte) *Year {
```

Reply:
I used to return extra bool, because of easier check it is true, but for this small program it makes sense to use as simple syntax as possible